packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -231,16 +231,16 @@ class BottomNavigationBar extends StatefulWidget {
   /// The [iconSize], [selectedFontSize], [unselectedFontSize], and [elevation]
   /// arguments must be non-null and non-negative.
   ///
-  /// If [selectedLabelStyle.color] and [unselectedLabelStyle.color] values
+  /// If [selectedLabelStyle].color and [unselectedLabelStyle].color values
```

Comment:
Ahh, so we can't refer to members, except by type?  So `[TextStyle.color]` works, but not `[selectedLabelStyle.color]`, right?

Reply:
Correct.  Dartdoc does have enough information to figure this out, but it would be an additional feature.  Previously we just pretended it worked...